domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -225,4 +225,21 @@ public String toString() {
         return ReflectionToStringBuilder.toString(this, ourStyle());
     }
 
+    @JsonProperty("mediationDeadline")
+    public Optional<LocalDate> getMediationDeadline() {
+        return Optional.ofNullable(this.respondedAt)
+            .map(LocalDateTime::toLocalDate)
+            .map(date -> date.plusDays(5))
+            .map(NextWorkingDay::new)
```

Comment:
why are we mapping to NextWorkingDay 🤔 - this doesn't actually apply any next working day logic does it?

Reply:
this is non java 8 way of writing it - `            return Optional.ofNullable(new NextWorkingDay(this.respondedAt.toLocalDate().plusDays(5)).getDate());
`

Not sure how i can explain, i will try - getDate uses a class variable which is set in NextWorkingDay. So, i think i have to do it like the way it is..